DigitalLearningSolutions.Web/Controllers/MyAccountController.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;

```

Comment:
Are these used? It might be worth doing a quick Cleanup Code for auto-formatting the files you've touched :) (we're currently not running any formatting checks on pipeline)